src/chrome/content/rules/Infobae.com.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Nonfunctional hosts in *.infobae.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Infobae.com">
+	<target host="www.infobae.com" />
+
+	<securecookie host="^\w" name=".+" />
+
```

Comment:
You can add this additional rule `<rule from="^http://infobae\.com/" to="https://www.infobae.com/" />` so that http://infobae.com gets redirected to https://www.infobae.com